db/queries/application/queries.py

```diff
@@ -15,13 +17,17 @@
 from external_services.aws import FileData
 from external_services.aws import list_files_by_prefix
 from flask import current_app
+from fsd_utils import extract_questions_and_answers
+from fsd_utils import generate_text_of_application
 from sqlalchemy import func
 from sqlalchemy import select
 from sqlalchemy.exc import IntegrityError
 from sqlalchemy.orm import joinedload
 from sqlalchemy.orm import noload
 from sqlalchemy.sql.expression import Select
 
+DOCUMENT_UPLOAD_SIZE_LIMIT = 2 * 1024 * 1024
```

Comment:
Should this be moved to the config? (Only reason is if we plan to change this)

Reply:
Fixed